examples/telegram/responses/callback_queries.py

```diff
@@ -0,0 +1,149 @@
+            TRANSITIONS: {
+                ("general", "keyboard"): messenger.cnd.message_handler(
+                    commands=["start", "restart"]
+                )
+            },
+        },
+    },
+    "general": {
+        "keyboard": {
+            RESPONSE: Response(
+                **{
+                    "text": "Starting test! What's 9 + 10?",
+                    # Here, we use a generic keyboard class.
+                    # Compare with the next script node.
+                    "ui": Keyboard(
+                        buttons=[Button(text="19", payload="19"), Button(text="21", payload="21")]
+                    ),
+                }
+            ),
+            TRANSITIONS: {
+                ("general", "native_keyboard"): messenger.cnd.callback_query_handler(
+                    func=lambda call: call.data == "19"
+                ),
+                ("general", "fail"): messenger.cnd.callback_query_handler(
+                    func=lambda call: call.data == "21"
+                ),
+            },
+        },
+        "native_keyboard": {
+            RESPONSE: Response(
+                **{
+                    "text": "Question: What's 2 + 2?",
+                    # In this case, we use telegram-specific classes.
+                    # They derive from the generic ones and include more options,
+                    # e.g. simple keyboard or inline keyboard.
+                    "ui": TelegramUI(
+                        buttons=[
+                            TelegramButton(text="5", payload="5"),
+                            TelegramButton(text="4", payload="4"),
+                        ],
+                        is_inline=False,
+                        row_width=4,
+                    ),
+                }
+            ),
+            TRANSITIONS: {
+                ("general", "success", 1.2): messenger.cnd.message_handler(
+                    func=lambda msg: msg.text == "4"
+                ),
+                ("general", "fail", 1.0): cnd.true(),
+            },
+        },
+        "success": {
+            RESPONSE: Response(
+                **{"text": "Success!", "ui": TelegramUI(keyboard=types.ReplyKeyboardRemove())}
+            ),
+            TRANSITIONS: {("root", "fallback"): cnd.true()},
+        },
+        "fail": {
+            RESPONSE: Response(
+                **{
+                    "text": "Incorrect answer, type anything to try again",
+                    "ui": TelegramUI(keyboard=types.ReplyKeyboardRemove()),
+                }
+            ),
+            TRANSITIONS: {("general", "keyboard"): cnd.true()},
+        },
+    },
+}
+
+interface = PollingTelegramInterface(messenger=messenger)
+
+
+# %%
+pipeline = Pipeline.from_script(
+    script=script,
+    start_label=("root", "start"),
+    fallback_label=("root", "fallback"),
+    context_storage=dict(),
+    messenger_interface=interface,
+)
+
+if __name__ == "__main__":
```

Comment:
See comment to `examples/telegram/interfaces/polling.py`.

Reply:
Replied above